docs/blazingsql/sql_reference/data_types.ipynb

```diff
@@ -166,9 +166,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "RAPIDS Stable",
+   "display_name": "Python [conda env:bsql5] *",
```

Comment:
I think this might cause problems when run in the app. We should revert it back to RAPIDS Stable